plugins/rikki/loungeviews/components/TimelineEntries.php

```diff
@@ -107,7 +107,10 @@ public function onRender()
                     break;
                 case 'division':
                     $timeline = new Collection([]);
-                    Timeline::with('divisions', 'matches.teams.divisions', 'teams.divisions', 'sloths.teams', 'teams.logo')->orderBy('created_at', 'desc')->chunk(100, function ($someTimelines) use ($id, $timeline) {
+                    Timeline::with('divisions', 'matches.teams.divisions', 'teams.divisions', 'sloths.teams.divisions', 'teams.logo')
+                            ->orderBy('created_at', 'desc')->chunk(100, function ($someTimelines) use ($id, $timeline) {
+
+
```

Comment:
nit: Never two empty lines at once (and here I think none since we are just starting a block of code)